src/main/io/osd.c

```diff
@@ -922,6 +920,17 @@ void osdUpdate(timeUs_t currentTimeUs)
         iterationCounter = 0;
         osdRefreshStats(currentTimeUs);
     }
+    if (refreshTimeout) {
+        if (checkStickPosition(THR_HI) || checkStickPosition(PIT_HI)) { // hide statistics
+            refreshTimeout = 1;
+        }
+        refreshTimeout--;
+        if (!refreshTimeout) {
+            max7456ClearScreen();
+        }
```

Comment:
You miss one line here:

max7456DrawScreenPartial();

before return without this statistics/spash is not displayed correctly.